pkgpanda/build/__init__.py

```diff
@@ -778,8 +778,18 @@ def get_build_ids(self):
 
 
 def build(package_store, name, variant, clean_after_build, recursive=False):
+    variant_str = pkgpanda.util.variant_str(variant)
```

Comment:
Should use https://github.com/dcos/dcos/blob/master/pkgpanda/util.py#L33 / variant_name in this printing. That will always make a non-empty string, and should always be printed.